pycbc/types/array.py

```diff
@@ -615,6 +615,19 @@ def conj(self):
     def squared_norm(self):
         """ Return the elementwise squared norm of the array """
 
+
+    # Adding the @_checkother decorator here raises a bizarre failure when
+    # calling weave. KeyError: 'V'. Why??
```

Comment:
Not sure on the 'v' keyerror, but I think that checkother expects only one other argument, and so won't work automatically here.